tools/pre-commit/shell_check_no_set_x.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Verify no set -x is used in this repository
+
+files="$(git grep -rnE --color=always 'set( +-[^ ])* +-[^ ]*(x|( +xtrace))' -- ':*.sh' ':*/Dockerfile' ':*.yaml' ':*.yml' ":(exclude)$0" ':(exclude).pre-commit-config.yaml')"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Can you add an example of line caught by this regex? Also having the regex isolated in the comment, can help testing it in https://regexr.com/ to understand how it works

Reply:
Thank you!